src/include/daos/tests_lib.h

```diff
@@ -166,6 +169,20 @@ struct dts_context {
 	/** OUTPUT END */
 };
 
+
+static inline bool
+tsc_create_pool(struct dts_context *tsc)
+{
+	return !tsc->tsc_skip_pool_create;
+}
+
+static inline bool
+tsc_create_cont(struct dts_context *tsc)
+{
+	/* Can't skip container if pool isn't also skipped */
+	return tsc_create_pool(tsc) && !tsc->tsc_skip_cont_create;
```

Comment:
"tsc_create_pool(tsc) || !tsc->tsc_skip_cont_create;" ?

Reply:
I don't think so. To create a container we want both conditions to be true (creating a pool AND not skipping container create). Unless I'm missing something??

Reply:
As my understand, if the pool needs to be created, then the container also needs to be created in spite of whether you specify the container UUID or not. If the pool needs NOT be created, then whether creating the container depends on whether specify the container UUID or not.

Reply:
Yes, you're right. I'll update it.